javatests/arcs/android/storage/database/DatabaseImplTest.kt

```diff
@@ -480,6 +542,26 @@ class DatabaseImplTest {
         assertThat(entityOut).isEqualTo(entity)
     }
 
+    @Test
+    fun insertAndGet_entity_withCreationAndExpiration() = runBlockingTest {
```

Comment:
It looks like `expiration_timestamp` is optional? If so, could you add a test case for that, where you don't set it?

If it's not optional, then you probably need to update the SQL CREATE statement, since `expiration_timestamp` is nullable there.

Reply:
actually expiration_timestamp defaults to -1 in RawEntity, so it's not nullable. this particular test verifies the case when it is set to an actual value. other existing tests already cover the no explicit expiration cases.
I also added the missing `NOT NULL` to DatabaseImpl